uberpoet/util.py

```diff
@@ -59,6 +59,19 @@ def merge_lists(two_d_list):
     return [i for li in two_d_list for i in li]
 
 
+def percentage_split(seq, percentages):
```

Comment:
Sorry to bring it up again, but can you cite what stack overflow link you copy connivence functions like this in the future in general?  In this case it's https://stackoverflow.com/questions/14280856/separate-a-list-into-four-parts-based-on-percentage-even-if-the-list-is-not-divi

Also could you add this rule to the CONTRIBUTING.md file?

Reply:
yeap!

Reply:
yep will do

Reply:
updated with my own. The implementation is relatively trivial to be honest. I definitely did not need the `yield` or other stuff but logic remains similar. Keep track of an index that is calculated to split the list up to a given percentage and then slice the list and store it in an array of results.